### ISSUE TITLE
Yellow error message box too wordy

### ISSUE THREAD

**Issue by rkenyon** [PATRIC3/patric3_website#711]
3.1.118alpha
When selecting data sets are too voluminous, the message is a bit long and confusing.  Suggest cutting it down to something like "Some Tabs disabled due to data volume ...click for more information" 

![screen shot 2016-07-26 at 9 59 50 am](https://cloud.githubusercontent.com/assets/3188570/17143030/bc8364cc-5317-11e6-83cb-b564a8327165.png)

**Comment by dmachi** [comment]
This box no longer exists, there is new text which is probalby too terse at the moment.
